content/en/bits_ai/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Bits AI
+kind: documentation
+disable_toc: false
+private: true
+is_beta: true
+further_reading:
+    - link: "https://www.datadoghq.com/product/platform/bits-ai/"
+      tag: "Product page"
+      text: "Bits AI"
+    - link: "https://www.datadoghq.com/blog/datadog-bits-generative-ai/"
+      tag: "Blog post"
+      text: "Introducing Bits AI, Your New DevOps Copilot"
+    - link: "https://www.datadoghq.com/about/latest-news/press-releases/datadog-announces-bits-an-ai-assistant-to-help-engineers-quickly-resolve-application-issues/"
+      tag: "Press release"
+      text: "Bits AI announcement"
+---
+
+<div class="alert alert-info">Bits AI is in private beta.</div>
```

Comment:
Fill out this [form](dashcon.io/bits-ai) to join the waitlist.

Reply:
@kaixintai That link doesn't seem to be working for me, can you try again?